static/src/js/util/map/normalizeGranuleSpatial.js

```diff
@@ -83,19 +166,25 @@ const normalizeGranuleSpatial = (granule) => {
       const [swLat, swLon, neLat, neLon] = box.split(' ').map((coord) => parseFloat(coord))
 
       // Create a polygon from the bounding box
-      const polygonCoordinates = [
+      // TODO make sure this is counter-clockwise
```

Comment:
```suggestion
      // TODO make sure this is counter-clockwise
```
still needed?

Reply:
You can remove those in your branch since you are working in that file